biogrid/libs/biogrid-simulator/src/lib/biobrain/biobrain.spec.ts

```diff
@@ -13,9 +13,24 @@ const name3 = `${GRID_ITEM_NAMES.ENERGY_USER}-3`;
 beforeAll(() => {
   actualBrain = BioBrain.Instance;
   const ruralArea = [
-    new Building(BUILDING.MAX_CAPACITY, 3, 4, name1),
-    new Building(0, 7, 9, name2),
-    new Building(BUILDING.MAX_CAPACITY, 7, 8, name3),
+    new Building({
+      energy: BUILDING.MAX_CAPACITY,
+      x: 3,
+      y: 4,
+      gridItemName: name1
```

Comment:
I know it's not part of this PR, but these constants should be uppercase `const MY_GLOBAL_CONSTANT = 'abc';`

Reply:
I will address this